manual_tests/heartbeat_test/heartbeat_test.c

```diff
@@ -46,49 +46,58 @@ int main() {
         hb_ping_period_s = 25;
     }
 
-#ifdef PING_PERIOD
-    // optional override
-    hb_ping_period_s = PING_PERIOD;
-#endif
+    // optional override for ping period
+    if (PING_PERIOD){
+        hb_ping_period_s = PING_PERIOD;
+    }
 
-#ifndef IGNORE_PINGS
-    print("Initializing heartbeat...\n");
-    init_hb(SELF_ID);
-    print("Done init\n");
-#else
-    print("Skipped initializing heartbeat\n");
-#endif
+    if (!IGNORE_PINGS){
+        print("Initializing heartbeat...\n");
+        init_hb(SELF_ID);
+        print("Done init\n");
+    }
+    else {
+        print("Skipped initializing heartbeat\n");
+    }
 
     print("hb_self_id = ");
     if (hb_self_id == HB_OBC) {
-        print("OBC");
+        print("OBC\n");
     } else if (hb_self_id == HB_EPS) {
-        print("EPS");
+        print("EPS\n");
     } else if (hb_self_id == HB_PAY) {
-        print("PAY");
+        print("PAY\n");
     }
-    print("\n");
 
     print("hb_ping_period_s = %lu\n", hb_ping_period_s);
 
     print("Starting main loop\n");
 
-    while (1) {
-
-#ifdef DEBUG_LOG
-        print_bool("hb_send_obc_ping", hb_send_obc_ping);
-        print_bool("hb_send_eps_ping", hb_send_eps_ping);
-        print_bool("hb_send_pay_ping", hb_send_pay_ping);
-        print_bool("hb_received_obc_resp", hb_received_obc_resp);
-        print_bool("hb_received_eps_resp", hb_received_eps_resp);
-        print_bool("hb_received_pay_resp", hb_received_pay_resp);       
-        print_bool("hb_send_obc_resp", hb_send_obc_resp);
-        print_bool("hb_send_eps_resp", hb_send_eps_resp);
-        print_bool("hb_send_pay_resp", hb_send_pay_resp);
-#endif
 
+    while (1) {
```

Comment:
Need some way in this test file to initiate a request for the restart information